pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -215,6 +215,16 @@ func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit
 		resources.CpuPeriod = cpuPeriod
 	}
 
+	// the "memory.oom.group" flag appeared in kernel 4.19 and the docs indicate that 5.8 is the minimum kernel version
+	// if using cgroups v2, so we just need to check for that
```

Comment:
In addition, `runc` does not allow resources.Unified for cgroup v1, so cgroup v2 is a hard requirement here.

Maybe you can rephrase your comment based on that fact.